lms/djangoapps/mobile_api/models.py

```diff
@@ -25,3 +26,63 @@ def get_video_profiles(cls):
         Get the list of profiles in priority order when requesting from VAL
         """
         return [profile.strip() for profile in cls.current().video_profiles.split(",") if profile]
+
+
+class MobileAppVersionConfig(ConfigurationModel):  # pylint: disable=model-missing-unicode
+    """
+    Configuration for mobile app versions available.
+    """
+    IOS = "ios"
+    ANDROID = "android"
+    MOBILE_PLATFORM = (
+        (IOS, "IOS"),
+        (ANDROID, "ANDROID"),
+
+    )
+    KEY_FIELDS = ('platform', )  # mobile platform is unique
+    platform = CharField(
+        max_length=50,
+        choices=MOBILE_PLATFORM,
+        blank=False,
+        help_text="mobile device platform")
+    latest_version = TextField(blank=False, help_text="Latest available version for app in X.X.X format")
+    min_supported_version = TextField(blank=False, help_text="Min supported version for app in X.X.X format")
+    next_min_supported_version = TextField(
```

Comment:
We do not need 2 `min_supported_version`s.  Only the `latest_version` and `min_supported_version` should be sufficient.  Adding more will make it hard to maintain and makes the logic more complicated.